### ISSUE TITLE
RFTools Modular Stotare

### ISSUE THREAD

**Issue by turtleman8787** [AllTheMods/ATM-6#427]
Quick sort command when in the modular storage and the storage module inside disappears along with all items it contains

**To Reproduce**
Steps to reproduce the behaviour:

Use the quick sort command when in the modular storage and the storage module inside disappears along with all items it contains. Happens every single time you try and sort your inventory or the storage module's inventory 

**Expected behavior**
The storage modual gets deleted along with all items inside

**Screenshots**

**Additional context**

**Comment by ViRb3** [comment]
#327

**Comment by dananuch** [comment]
I saw the same issue.  I tried it when my inventory and modular storage were not full -  so there was room for everything.  I then attempted the quick "Middle mouse button" Sort.  I didn't loose anything.  I think its the Garbage can in the Inventory space that causes the stuff to get deleted.  Just a thought no real ability to validate that.  The sort moved everything in Modular storage to my inventory which would also be a problem too - I just wanted to sort the stuff in the modular unit.  Just a bit more data for anyone looking to figure it out.
